# HG changeset patch
# User Michael Godfrey <michaeldgodfrey@gmail.com>
# Date 1380557035 14400
#      Mon Sep 30 12:03:55 2013 -0400
# Node ID 3670ce8da1a06808a4fd29fcc5decfad5d8a5f7b
# Parent  8a303b01cdd06d7b106137b273d6f69a6b3f1cba
doc: improve format of table 15.1 for TeX characters.

* doc/interpreter/plot.txi: improve format of table 15.1 for TeX characters.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -634,36 +634,41 @@ Finally, the superscript and subscriptin
 and @qcode{'_'} characters.  If the @qcode{'^'} or @qcode{'_'} is followed by a
 @{ character, then all of the block surrounded by the @{ @} pair is super- or
 sub-scripted.  Without the @{ @} pair, only the character immediately following
 the @qcode{'^'} or @qcode{'_'} is super- or sub-scripted.
 
 @float Table,tab:extended
 @tex
 \vskip 6pt
-\def\symtable#1#2#3{\hbox to \hsize {\hfill\vbox{\offinterlineskip \tabskip=0pt
+\newdimen\cola \cola=78pt
+\newdimen\colb \colb=78pt
+\newdimen\colc \colc=78pt
+\def\symtable#1#2#3{
+\hbox to \hsize {\hfill\vbox{\offinterlineskip \tabskip=0pt
 \hskip36pt #1
 \vskip6pt
 \halign{
 \vrule height2.0ex depth1.ex width 0.6pt #2\tabskip=0.3em &
 #2 \hfil & \vrule #2 & #2 \hfil & #2 \vrule &
 #2 \hfil & \vrule #2 & #2 \hfil & #2 \vrule &
 #2 \hfil & \vrule #2 & #2 \hfil & #2 \vrule
 width 0.6pt \tabskip=0pt\cr
 \noalign{\hrule height 0.6pt}
 & Code && Sym && Code && Sym && Code && Sym &\cr
 \noalign{\hrule}
 #3
 \noalign{\hrule height 0.6pt}
 }
 }\hfill}}
+\hoffset72pt
 \symtable{Greek Lowercase Letters} {#}
-{& $\backslash$alpha     && $\alpha$
-&& $\backslash$beta      && $\beta$
-&& $\backslash$gamma     && $\gamma$     &\cr
+{& \hbox to \cola{$\backslash$alpha }    && $\alpha$
+&& \hbox to \colb{$\backslash$beta }     && $\beta$
+&& \hbox to \colc{$\backslash$gamma}     && $\gamma$     &\cr
 & $\backslash$delta      && $\delta$
 && $\backslash$epsilon   && $\epsilon$
 && $\backslash$zeta      && $\zeta$      &\cr
 & $\backslash$eta        && $\eta$
 && $\backslash$theta     && $\theta$
 && $\backslash$vartheta  && $\vartheta$  &\cr
 & $\backslash$iota       && $\iota$
 && $\backslash$kappa     && $\kappa$
@@ -680,33 +685,33 @@ width 0.6pt \tabskip=0pt\cr
 & $\backslash$tau        && $\tau$
 && $\backslash$upsilon   && $\upsilon$
 && $\backslash$phi       && $\phi$       &\cr
 & $\backslash$chi        && $\chi$
 && $\backslash$psi       && $\psi$
 && $\backslash$omega     && $\omega$     &\cr}
 \vskip12pt
 \symtable{Greek Uppercase Letters} {#}
-{& $\backslash$Gamma   && $\Gamma$
-&& $\backslash$Delta   && $\Delta$
-&& $\backslash$Theta   && $\Theta$      &\cr
+{& \hbox to \cola{$\backslash$Gamma}   && $\Gamma$
+&& \hbox to \colb{$\backslash$Delta}   && $\Delta$
+&& \hbox to \colc{$\backslash$Theta}   && $\Theta$      &\cr
 & $\backslash$Lambda   && $\Lambda$
 && $\backslash$Xi      && $\Xi$
 && $\backslash$Pi      && $\Pi$         &\cr
 & $\backslash$Sigma    && $\Sigma$
 && $\backslash$Upsilon && $\Upsilon$
 && $\backslash$Phi     && $\Phi$        &\cr
 & $\backslash$Psi      && $\Psi$
 && $\backslash$Omega   && $\Omega$
 &&    &&       &\cr}
 \vskip12pt
 \symtable{Misc Symbols Type Ord} {#}
-{& $\backslash$aleph       && $\aleph$
-&& $\backslash$wp          && $\wp$
-&& $\backslash$Re          && $\Re$          &\cr
+{& \hbox to \cola{$\backslash$aleph}       && $\aleph$
+&& \hbox to \colb{$\backslash$wp}          && $\wp$
+&& \hbox to \colc{$\backslash$Re}          && $\Re$      &\cr
 & $\backslash$Im           && $\Im$
 && $\backslash$partial     && $\partial$
 && $\backslash$infty       && $\infty$       &\cr
 & $\backslash$prime        && $\prime$
 && $\backslash$nabla       && $\nabla$
 && $\backslash$surd        && $\surd$        &\cr
 & $\backslash$angle        && $\angle$
 && $\backslash$forall      && $\forall$
@@ -714,24 +719,24 @@ width 0.6pt \tabskip=0pt\cr
 & $\backslash$neg          && $\neg$
 && $\backslash$clubsuit    && $\clubsuit$
 && $\backslash$diamondsuit && $\diamondsuit$ &\cr
 & $\backslash$heartsuit    && $\heartsuit$
 && $\backslash$spadesuit   && $\spadesuit$
 &&    &&       &\cr}
 \vskip12pt
 \symtable{``Large'' Operators} {#}
-{& $\backslash$int   && $\int$
-&&    &&
-&&    &&       &\cr}
+{& \hbox to \cola{$\backslash$int}   && $\int$
+&& \hbox to \colb{}   &&
+&& \hbox to \colc{}   &&       &\cr}
 \vskip12pt
 \symtable{Binary operators} {#}
-{& $\backslash$pm     && $\pm$
-&& $\backslash$cdot   && $\cdot$
-&& $\backslash$times  && $\times$      &\cr
+{& \hbox to \cola{$\backslash$pm}     && $\pm$
+&& \hbox to \colb{$\backslash$cdot}   && $\cdot$
+&& \hbox to \colc{$\backslash$times}  && $\times$      &\cr
 & $\backslash$ast     && $\ast$
 && $\backslash$circ   && $\circ$
 && $\backslash$bullet && $\bullet$     &\cr
 & $\backslash$div     && $\div$
 && $\backslash$cap    && $\cap$
 && $\backslash$cup    && $\cup$        &\cr
 & $\backslash$vee     && $\vee$
 && $\backslash$wedge  && $\wedge$
@@ -792,80 +797,84 @@ width 0.6pt \tabskip=0pt\cr
 @item @tab  \deg
 @end multitable
 @end ifnottex
 @caption{Available special characters in @TeX{} mode}
 @end float
 @float
 @tex
 \vskip 6pt
+\newdimen\cola \cola=78pt
+\newdimen\colb \colb=78pt
+\newdimen\colc \colc=78pt
 \def\symtable#1#2#3{\hbox to \hsize {\hfill\vbox{\offinterlineskip \tabskip=0pt
 \hskip36pt #1
 \vskip6pt
 \halign{
 \vrule height2.0ex depth1.ex width 0.6pt #2\tabskip=0.3em &
 #2 \hfil & \vrule #2 & #2 \hfil & #2 \vrule &
 #2 \hfil & \vrule #2 & #2 \hfil & #2 \vrule &
 #2 \hfil & \vrule #2 & #2 \hfil & #2 \vrule
 width 0.6pt \tabskip=0pt\cr
 \noalign{\hrule height 0.6pt}
 & Code && Sym && Code && Sym && Code && Sym &\cr
 \noalign{\hrule}
 #3
 \noalign{\hrule height 0.6pt}
 }
 }\hfill}}
-\vskip6pt
+\hoffset72pt
+\vskip12pt
 \symtable{Relations} {#}
-{& $\backslash$leq      && $\leq$
-&& $\backslash$subset   && $\subset$
-&& $\backslash$subseteq && $\subseteq$    &\cr
+{& \hbox to \cola{$\backslash$leq}      && $\leq$
+&& \hbox to \colb{$\backslash$subset}   && $\subset$
+&& \hbox to \colc{$\backslash$subseteq} && $\subseteq$    &\cr
 & $\backslash$in        && $\in$
 && $\backslash$geq      && $\geq$
 && $\backslash$supset   && $\supset$      &\cr
 & $\backslash$supseteq  && $\supseteq$
 && $\backslash$ni       && $\ni$
 && $\backslash$mid      && $\mid$         &\cr
 & $\backslash$equiv     && $\equiv$
 && $\backslash$sim      && $\sim$
 && $\backslash$approx   && $\approx$      &\cr
 & $\backslash$cong      && $\cong$
 && $\backslash$propto   && $\propto$
 && $\backslash$perp     && $\perp$        &\cr}
-\vskip6pt
+\vskip12pt
 \symtable{Arrows} {#}
-{& $\backslash$leftarrow      && $\leftarrow$
-&& $\backslash$Leftarrow      && $\Leftarrow$
-&& $\backslash$rightarrow     && $\rightarrow$      &\cr
+{& \hbox to \cola{$\backslash$leftarrow}      && $\leftarrow$
+&& \hbox to \colb{$\backslash$Leftarrow}      && $\Leftarrow$
+&& \hbox to \colc{$\backslash$rightarrow}     && $\rightarrow$      &\cr
 & $\backslash$Rightarrow      && $\Rightarrow$
 && $\backslash$leftrightarrow && $\leftrightarrow$
 && $\backslash$uparrow        && $\uparrow$         &\cr
 & $\backslash$downarrow       && $\downarrow$
-&&   &&
+&&   && 
 &&   &&       &\cr}
 \vskip12pt
 \symtable{Openings and Closings} {#}
-{& $\backslash$lfloor   && $\lfloor$
-&& $\backslash$langle   && $\langle$
-&& $\backslash$lceil    && $\lceil$      &\cr
+{& \hbox to \cola{$\backslash$lfloor   }    && $\lfloor$
+&& \hbox to \colb{$\backslash$langle   }    && $\langle$
+&& \hbox to \colc{$\backslash$lceil    }    && $\lceil$      &\cr
 & $\backslash$rfloor    && $\rfloor$
 && $\backslash$rangle   && $\rangle$
 && $\backslash$rceil    && $\rceil$      &\cr}
 \vskip12pt
 \symtable{Alternate Names} {#}
-{& $\backslash$neq   && $\neq$
-&&   &&
-&&   &&   &\cr}
+{& \hbox to \cola{$\backslash$neq}   && $\neq$
+&& \hbox to \colb{}   && 
+&& \hbox to \colc{}   &&   &\cr}
 \vskip12pt
 \symtable{Other (not in Appendix F Tables)} {#}
-{& $\backslash$ldots     && $\ldots$
-&& $\backslash$0         && $\oslash$
-&& $\backslash$copyright && $\copyright$      &\cr
+{& \hbox to \cola{$\backslash$ldots}     && $\ldots$
+&& \hbox to \colb{$\backslash$0}         && $\oslash$
+&& \hbox to \colc{$\backslash$copyright} && $\copyright$      &\cr
 & $\backslash$deg        && $\deg$
-&&    &&
+&&    &&  
 &&    &&       &\cr}
 \vskip12pt
 \hskip36pt Table 15.1: Available special characters in \TeX\ mode (cont.)
 @end tex
 @end float
 
 A complete example showing the capabilities of the extended text is
 
